### PR TITLE
Fix repo reference

### PR DESCRIPTION
**Pull request by futuretea**
**Related Issue:**
#727

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ steps:
     dockerfile: package/Dockerfile
     password:
       from_secret: docker_password
-    repo: "rancher/harvester"
+    repo: "harvester/harvester"
```
Comment (guangbochen) [inline .drone.yml]: The docker repo is not changed due to rate limiting, so we don't need to change it.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Rancher Harvester
 ========
-[![Build Status](https://drone-publish.rancher.io/api/badges/rancher/harvester/status.svg)](https://drone-publish.rancher.io/rancher/harvester)
-[![Go Report Card](https://goreportcard.com/badge/github.com/rancher/harvester)](https://goreportcard.com/report/github.com/rancher/harvester)
+[![Build Status](https://drone-publish.rancher.io/api/badges/harvester/harvester/status.svg)](https://drone-publish.rancher.io/harvester/harvester)
```
Comment (gitlawr) [inline README.md:3]: The URL may be invalid. Revisit this when CI is ready

File: pkg/controller/master/upgrade/upgrade_controller.go [hunk 3]
```diff
@@ -24,7 +24,7 @@ const (
 	harvesterManagedLabel          = "harvesterhci.io/managed"
 	harvesterLatestUpgradeLabel    = "harvesterhci.io/latestUpgrade"
 	harvesterUpgradeComponentLabel = "harvesterhci.io/upgradeComponent"
-	upgradeImageRepository         = "rancher/harvester-bundle"
+	upgradeImageRepository         = "harvester/harvester-bundle"
```
Comment (gitlawr) [inline pkg/controller/master/upgrade/upgrade_controller.go]: Keep the image repository unchanged

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**